### PR TITLE
fix issue 594 - enable sharding for chunkstore

### PR DESCRIPTION
**Pull request by bmoscon**
see #594

### CODE REVIEW

File: arctic/chunkstore/chunkstore.py [hunk 1]
```diff
@@ -43,8 +43,14 @@
 
 class ChunkStore(object):
     @classmethod
-    def initialize_library(cls, arctic_lib, **kwargs):
+    def initialize_library(cls, arctic_lib, hashed=True, **kwargs):
         ChunkStore(arctic_lib)._ensure_index()
+        
+        logger.info("Trying to enable sharding...")
+        try:
+            enable_sharding(arctic_lib.arctic, arctic_lib.get_name(), hashed=hashed)
```
[the remark this answers is not in the document]
Comment (bmoscon) [reply arctic/chunkstore/chunkstore.py]: oh yeah really good catch. the default is the string symbol, but chunkstore uses `sy`, not `symbol` like version store

[the remark this answers is not in the document]
Comment (bmoscon) [reply arctic/chunkstore/chunkstore.py]: take another look @dimosped I've updated it

[the remark this answers is not in the document]
Comment (dimosped) [reply arctic/chunkstore/chunkstore.py]: looks great @bmoscon

**Comment by bmoscon** [comment]
thanks @dimosped 

I'll give @jamesblackburn a chance to look at it, otherwise I'll merge it in later today or tomorrow

**Pull request merged**